Logos/WireX_Logo_Dark.svg

```diff
@@ -0,0 +1,409 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
As we discussed on Friday, please follow logo guidelines in the step 5.3 of the data connectors documentation https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors#build-the-connector - pasting that text below as well. This logo file has xlink_href and xlink either of which will fail to deploy the logo file. refer to https://github.com/Azure/Azure-Sentinel/blob/master/Logos/OrcaSecurity.svg?short_path=44fa7f3 as an example of a well formed logo file.

The company logo adhering to the following requirements in the 'Logo' folder
Logo needs to be in SVG format and under 5 Kb
Ensure raw file of logo does not have any of the following:
cls and style formats
embedded png formats
xmlns:xlink
data-name
Do not use xlink:href - use inline instead
Do not use title tag
If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets
Logo scales well to fit in a 75 px square
SVG code file is formatted well for readability